cloud: add release notes; update roadmap (#12692)
pingcap/docs: tidb-cloud/tidb-cloud-release-notes.md

This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.

## February 28, 2023

**General changes**

> - Add the [SQL Diagnosis](/tidb-cloud/tune-performance.md) feature for Serverless Tier clusters.

Reviewer: ```suggestion
- Add the [SQL Diagnosis](/tidb-cloud/tune-performance.md) feature for [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters.
```

>     With SQL Diagnosis, you can gain deep insights into SQL-related runtime status, which makes the SQL performance tuning more efficiently. Currently, the SQL Diagnosis feature for Serverless Tier only provides slow query data.

Author: ```suggestion
    With SQL Diagnosis, you can gain deep insights into SQL-related runtime status, which makes the SQL performance tuning more efficient. Currently, the SQL Diagnosis feature for Serverless Tier only provides slow query data.
```

>     With SQL Diagnosis, you can gain deep insights into SQL-related runtime status, which makes the SQL performance tuning more efficiently. Currently, the SQL Diagnosis feature for Serverless Tier only provides slow query data.

Reviewer: ```suggestion
    With SQL Diagnosis, you can gain deep insights into SQL-related runtime status, which makes the SQL performance tuning more efficient. Currently, the SQL Diagnosis feature for Serverless Tier only provides slow query data.
```

>     To use this feature, click **SQL Diagnosis** on the left navigation bar of your cluster page.

Reviewer 2: ```suggestion
    To use SQL Diagnosis, click **SQL Diagnosis** on the left navigation bar of your Serverless Tier cluster page.
```

> - Upgrade the default TiDB version of all [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters from [v6.4.0](https://docs.pingcap.com/tidb/v6.4/release-6.4.0) to [v6.6.0](https://docs.pingcap.com/tidb/v6.6/release-6.6.0).
> 

Author: ```suggestion
```

**Console changes**

- Optimize the left navigation.

    You can navigate pages more efficiently, for example:

>     - You can hover the mouse on the upper-left corner to quickly switch between clusters or projects.

Reviewer 2: ```suggestion
    - You can hover the mouse in the upper-left corner to quickly switch between clusters or projects.
```

    - You can switch between the **Clusters** page and the **Admin** page.

**API changes**

> - Release several TiDB Cloud API endpoints for the Import feature：

Reviewer 3: ```suggestion
- Release several TiDB Cloud API endpoints for data import：
```

>     - List Import Tasks
>     - Create Import Task
>     - Preview Data before Starting an Import Task
>     - Get Import Task Role Info
>     - Upload a Local File for an Import Task
>     - Get Import Task
>     - Update Import Task

Reviewer: ```suggestion
    - List all import tasks
    - Get an import task
    - Create an import task
    - Update an import task
    - Upload a local file for an import task
    - Preview data before starting an import task
    - Get the role information for import tasks
```

>     For more information, refer to the [API documentation](https://docs.pingcap.com/tidbcloud/api/v1beta#tag/Import).

Reviewer: ```suggestion
  For more information, refer to the [API documentation](https://docs.pingcap.com/tidbcloud/api/v1beta#tag/Import).
```
